web/src/composebox_typeahead.js

```diff
@@ -1170,78 +1186,90 @@ export function initialize({on_enter_send}) {
     $("form#send_message_form").on("keydown", (e) => handle_keydown(e, {on_enter_send}));
     $("form#send_message_form").on("keyup", handle_keyup);
 
-    bootstrap_typeahead.create($("input#stream_message_recipient_topic"), {
-        source() {
-            return topics_seen_for(compose_state.stream_id());
-        },
-        items: 3,
-        fixed: true,
-        highlighter_html(item) {
-            return typeahead_helper.render_typeahead_item({primary: item});
+    bootstrap_typeahead.create(
+        {
+            $element: $("input#stream_message_recipient_topic"),
+            type: "input",
```

Comment:
You can likely avoid the ugly re-indentation here by just doing a `const bootstrap_typeahead_input = ...` line just before this, rather than passing it inline. I'd recommend doing so.

Reply:
Great idea, did that